src/backend/core/api/serializers.py

```diff
@@ -27,6 +27,26 @@ class Meta:
         read_only_fields = ["id", "email", "full_name", "short_name"]
 
 
+class UserLightSerializer(serializers.ModelSerializer):
+    """Serialize users with limited fields."""
+
+    id = serializers.SerializerMethodField(read_only=True)
+    email = serializers.SerializerMethodField(read_only=True)
+
+    def get_id(self, _user):
+        """Return always None. Here to have the same fields than in UserSerializer."""
```

Comment:
Don't you want to inherit from `UserSerializer` then? Or is this temporary while the frontend is not ready?

Reply:
Done